### PR TITLE
✨ Speaker renaming & change

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

**Review by anuejn — CHANGES_REQUESTED** [review]
Im not really convinced that this is the right way to implement modals. Maybe we want to implement the API more similiar to how evergreen and other libraries do it?

File: frontend/src/app.tsx [hunk 1]
```diff
@@ -20,6 +21,7 @@ export function App() {
 
   return (
     <Router base={routerBase}>
+      <ModalHolder />
```
[the remark this answers is not in the document]
Comment (anuejn) [reply frontend/src/app.tsx:24]: I mean, it's a pretty nice hack :D

[the remark this answers is not in the document]
Comment (anuejn) [reply frontend/src/app.tsx:24]: I think I would be okay with it, what do you think @phlmn ?

[the remark this answers is not in the document]
Comment (pajowu) [reply frontend/src/app.tsx:24]: One advantage of doing this (which I broke nearly every time I used another library) is that this prevents us from opening 2 modals

[the remark this answers is not in the document]
Comment (pajowu) [reply frontend/src/app.tsx:24]: And tbh I don't think it really is much more hacky than what evergreen does https://github.com/segmentio/evergreen/blob/master/src/portal/src/Portal.js#L7

[the remark this answers is not in the document]
Comment (anuejn) [reply frontend/src/app.tsx:24]: I mean the portal feature is made specifically for these kinds of hacks. It just feels wiered to have a component do jsx-stuff in a somewhat side-effecty way

[the remark this answers is not in the document]
Comment (anuejn) [reply frontend/src/app.tsx:24]: But I do see the advantages of doing it like this

**Review by anuejn — APPROVED** [review]

**Pull request merged**